.github/workflows/prod-build.yml

```diff
@@ -182,6 +182,9 @@ jobs:
           REACT_APP_SURVEY_RATE_FROM_CSS_CASCADE_2022_B: 0.05
           REACT_APP_SURVEY_RATE_TILL_CSS_CASCADE_2022_B: 0.1
 
+          REACT_APP_GLEAN_CHANNEL: prod
+          REACT_APP_GLEAN_ENABLED: true
```

Comment:
Should we enable it in stage first before enabling it in prod? Will we see the data right away somewhere?

Reply:
Yes, once Glean is enabled and the related bug to enable it in the pipeline is fixed you'll be able to see the data right away.